django_restql/mixins.py

```diff
@@ -268,20 +280,16 @@ def fields(self):
             return {}
 
 
-class EagerLoadingMixin(object):
+class EagerLoadingMixin(RequestQueryParserMixin):
     @property
     def parsed_query(self):
         """
         Gets parsed query for use in eager loading. 
         Defaults to the serializer parsed query assuming
         using django-restql DynamicsFieldMixin.
         """
-        if hasattr(self, "get_serializer_class"):
-            serializer_class = self.get_serializer_class()
-
-            if issubclass(serializer_class, DynamicFieldsMixin):
-                if serializer_class.has_query_param(self.request):
-                    return serializer_class.get_parsed_query_from_req(self.request)
+        if self.has_query_param(self.request):
```

Comment:
I have concerns about using such a generic function name on a view. `has_query_param` could mean a lot of different things, especially if you aren't aware of the context. `has_restql_query_param` was what I went with on my PR to try and differentiate it and I'd like to recommend something like that here as well.

Reply:
`has_restql_query_param` is very long  and this is a public method, I don't think if it is a good idea to give it a long name, also `has_query_param(request)` makes sense with request arg, easy to guess that we are checking if the request has a query parameter.

Reply:
DRF provides `request.query_params`. My concern is that users will mistake this helper function as a way to check if query parameters were passed in, even if they are not `query` or our defined restql query parameter name.